src/utils/validators.js

```diff
@@ -13,8 +13,9 @@ const isPositiveNumber = function ( str ) {
 
 
 /** Returns true if a string represents a positive integer */
+/** @todo Change name to 'isWholeNumber'. */
 const isPositiveWholeNumber = function ( str ) {
-  return /^[0-9]*$/.test(str);
+  return str !== '' && /^[0-9]*$/.test(str);
 };
```

Comment:
Minor performance thing, but there's no reason to recreate the RegExp object every time you invoke the function.

Reply:
True! We should keep this in mind as an improvement. If you want to make a PR for it, I'd be into that. If not, we'll try to catch it later. Wish we could have a secondary list of issues that would stay out of sight without having to use search filters.

Reply:
Maybe an issue with just a list of minor improvements.

Reply:
You could use a project.

Reply:
Should definitely look into that (they're still a new feature to me).